README.md

```diff
@@ -1,4 +1,4 @@
-[![Tests](https://travis-ci.org/google/pytype.svg?branch=master)](https://travis-ci.org/google/pytype)
+![CI](https://github.com/google/pytype/workflows/CI/badge.svg?branch=master)(https://github.com/google/pytype/actions)
```

Comment:
Sorry I didn't catch this the first time around, but I believe you need an extra pair of brackets, like so:

<pre><b>[</b>![CI](https://github.com/google/pytype/workflows/CI/badge.svg?branch=master<b>]</b>(https://github.com/google/pytype/actions)</pre>

Right now, I'm seeing the second url displayed to the right of the badge: https://github.com/google/pytype/blob/98417a3dd47ef5947d7614cec87b8ccfa342140e/README.md

Reply:
my bad, didn't have a good markdown editor I guess.